app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluation.tsx

```diff
@@ -1155,6 +1155,12 @@ export default function Evaluation(props: EvaluationProps) {
                         colorscale: confusionMatrixConfig.log
                           ? confusionMatrix?.colorscale || "viridis"
                           : "viridis",
+                        hovertemplate:
+                          [
+                            "<b>count: %{z:d}</b>",
+                            `${evaluation?.info?.config?.gt_field}: %{y}`,
```

Comment:
Can you please update further like this:
https://github.com/voxel51/fiftyone/blob/8beca16fd9931f9262eef8b30e581510cef1f36b/fiftyone/core/plots/plotly.py#L137-L145

so that `truth` and `predicted` are used as default strings if the values don't exist for whatever reason?